tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py

```diff
@@ -10,18 +10,20 @@
 
 
 def test_create():
-    conf = EnsembleConfig()
+    with pytest.raises(ValueError):
+        EnsembleConfig()
+
+    conf = EnsembleConfig.from_dict({})
     assert len(conf) == 0
     assert "XYZ" not in conf
```

Comment:
Something which I think is a bad idea to have in the first place and that is, an internal implementation for:

```python
    def __contains__(self, key):
        return self._has_key(key)
```

Reply:
right, OK, just checks that it doesn't throw anything? that makes sense. i thought it might be trying to ascertain that ensemble config is empty, by checking one value :sweat_smile: